### ISSUE TITLE
Draft new page with both primary and general election audit and repayment process

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3066]
### Summary

**What we're after:**
This ticket is to draft content for a new page in our forthcoming bucket for public funding of presidential elections. This new page will cover the audit and repayment process in the general election.

After #2997 work was complete, consensus was to fit Primary and General audit/repayment content on the same page because of the similar regs are similar. There 

### Related issues

- #2997 (Primary election version)

### Completion criteria

- [x]  Identify content for General committees that should be added to the Primary draft
- [x]  Do legal research (Regs, E&Js, AOs) on general election audit and repayment process
- [x]   Identify approved language to use for page
- [x]   Draft content for page, noting all sources of language
- [x]   Circulate to content team
- [x]   Make any edits content team suggests

### Future work

- NEXT Ticket: Circulate to stakeholders in #3046.
- When stakeholder comments are back, create and publish the bucket collection page.
- Create new pages as child pages of the new bucket, and add links to collection page.
- Front end adds new bucket to H4CC hard-coded pages.

**Comment by bmathesonFEC** [comment]
Updated ticket to include completion criteria designed to fit Primary and General election guidance on the same page.

**Comment by bmathesonFEC** [comment]
Will circulate updated draft to Content team this afternoon.

**Comment by bmathesonFEC** [comment]
Although the regulations governing Primary vs. General election public funding are different, the Audit/Repayment sections are similar. As a result, updates required to make this page apply to primary and general funding were minimal.  I recirculated this to the Content Team for their comments.

**Comment by dorothyyeager** [comment]
@bmathesonFEC did a great job and our comments were minimal too. We'll include any needed edits in our circulation at #3046 so closing this ticket since drafting is complete.
